illumio_connector.py

```diff
@@ -476,29 +475,32 @@ def _handle_create_enforcement_boundary(self, param):
     def _handle_get_workloads(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
 
-        enforcement_mode = param.get("enforcement_mode").lower()
-        connectivity = param.get("connectivity")
+        enforcement_mode = param.get("enforcement_mode")
+        if enforcement_mode:
+            enforcement_mode = enforcement_mode.lower()
+            if enforcement_mode not in ENFORCEMENT_MODE_LIST:
+                return action_result.set_status(
+                    phantom.APP_ERROR,
+                    "Please enter a valid value for 'enforcement_mode' parameter",
+                )
+        online = param.get("online")
+        if online:
+            online = online == "True"
```

Comment:
Why are we not using the boolean data type?

Reply:
The reason for not using boolean here is because by default boolean value is "False" and as discussed we wanted the default value to be "None". Hence using dropdown here.